### PR TITLE
Fix typos in `CircularBufferTest.class.st`

### PR DESCRIPTION
**Pull request by citizen428**
I noticed two typos in this test file.

### CODE REVIEW

File: dev/src/Exercise@CircularBuffer/CircularBufferTest.class.st [hunk 1]
```diff
@@ -144,7 +144,7 @@ CircularBufferTest >> test05_FullBufferCantBeWrittenTo [
 
 	<exeTestName: 'full buffer can''t be written to'>
 	<exeTestUUID: '2af22046-3e44-4235-bfe6-05ba60439d38'>
-
```
Comment (citizen428) [inline dev/src/Exercise@CircularBuffer/CircularBufferTest.class.st:147]: All these whitespace changes where just auto-fixed by my editor, I can undo those if needed.

**Comment by citizen428** [comment]
Yup, my bad, fixed.

**Review by Bajger — CHANGES_REQUESTED** [review]
Hi! Actually, both places need to be changed. 
[dev/src/Exercise@CircularBuffer/CircularBufferTest.class.st](https://github.com/exercism/pharo-smalltalk/pull/625/files#diff-9387f24f0e801f88080d2c47a149087759524e7dff9a34569f204465aefd4a96)
and that you've originally changed: https://github.com/exercism/pharo-smalltalk/blob/main/exercises/practice/circular-buffer/CircularBufferTest.class.st

Usually I change first and then run generation to exercise directory, but you don't need to do that for small change. Both files should be in sync. Sorry for confusion.

**Comment by citizen428** [comment]
All good, done.

**Review by Bajger — APPROVED** [review]

**Comment by Bajger** [comment]
@citizen428 Thank you!

**Pull request merged**